xml/schema/Contact/Contact.xml

```diff
@@ -594,6 +595,7 @@
     <type>varchar</type>
     <length>255</length>
     <comment>Cache Postal greeting.</comment>
+    <readonly>true</readonly>
```

Comment:
These fields are actually UI selectable - ie when you go to edit them from contact summary you can choose a different option value

![image](https://user-images.githubusercontent.com/336308/110265906-88a93a00-8021-11eb-8dd1-5a5fec8da6d8.png)

Reply:
(Having trouble screenshotting the above in edit mode)

Reply:
Are yo sure you aren't thinking of the `postal_greeting_id` field?

Reply:
Looks like it - they are so clear those field too....